nf_core/lint_utils.py

```diff
@@ -73,7 +73,7 @@ def run_prettier_on_file(file):
     except subprocess.CalledProcessError as e:
         if ": SyntaxError: " in e.stdout.decode():
             raise ValueError(f"Can't format {file} because it has a synthax error.\n{e.stdout.decode()}") from e
```

Comment:
Proposed change as suggested by @awgymer 

```suggestion
            log.critical(f"Can't format {file} because it has a synthax error.\n{e.stdout.decode()}")
```

Reply:
@ewels & @awgymer you are welcome to commit this snippet, if this is what you had in mind.